packages/core/admin/webpack.alias.js

```diff
@@ -40,7 +40,7 @@ const aliasExactMatch = [
 // See https://webpack.js.org/configuration/resolve/
 module.exports = {
   ...aliasExactMatch.reduce((acc, name) => {
-    acc[`${name}$`] = require.resolve(name);
+    acc[`${name}$`] = path.resolve(__dirname, '..', '..', '..', 'node_modules', name);
```

Comment:
We probably want to use sth like findup to find the package up the tree if Strapi is in a monorepo senario